[E 2025-10-05 12:04:18,480 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:04:33,487 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:04:43,972 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:05:02,169 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:30:10,666 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:30:17,734 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:30:24,790 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
[E 2025-10-05 12:30:48,782 C:\Users\wildv\AppData\Local\Programs\Python\Python313\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 451}, 'response': {'token': '', 'accounts': [], 'detail': 'Too Many Connections.', 'ip_address': '125.231.148.209', 'person_id': 'K122373468', 'client_name': '', 'contract_download': False, 'web_token': '', 'solace_props': None}}
